tests/plugins/clipboard/dropimage.js

```diff
@@ -88,9 +107,51 @@
 			FileReader.setReadResult( 'load' );
 
 			setHtmlWithSelection( this.editor, '<p class="p">Paste image here:^</p>' );
-			assertDropImage( this.editor, dropEvt, imageType, expected, {
-				dropContainer: this.editor.editable().findOne( '.p' ).getChild( 0 ),
-				dropOffset: 17
+			assertDropImage( {
+				editor: this.editor,
+				event: dropEvt,
+				type: imageType,
+				expected: expected,
+				dropRange: {
+					dropContainer: this.editor.editable().findOne( '.p' ).getChild( 0 ),
+					dropOffset: 17
+				}
+			} );
+		},
+
+		// (#4750)
+		'test showing notification for dropping unsupported image type': function() {
```

Comment:
Maybe we could add our expectations to the test case?
```suggestion
		'test dropping unsupported image type shows notification': function() {
```